Dockerfile

```diff
@@ -215,14 +215,19 @@ ARG UAVS3D_COMMIT=073fe7cc8eaef021aa4adc7b0100461e0260b70d
 ARG RUBBERBAND_VERSION=2.0.0
 ARG RUBBERBAND_URL="https://breakfastquay.com/files/releases/rubberband-$RUBBERBAND_VERSION.tar.bz2"
 ARG RUBBERBAND_SHA256=eccbf0545496ce3386a2433ceec31e6576a76ed6884310e4b465003bfe260286
+# bump: libgme /LIBGME_COMMIT=([[:xdigit:]]+)/ gitrefs:https://bitbucket.org/mpyne/game-music-emu.git|re:#^refs/heads/master$#|@commit
+# bump: libgme after ./hashupdate Dockerfile LIBGME $LATEST
+# bump: libgme link "Source diff $CURRENT..$LATEST" https://bitbucket.org/mpyne/game-music-emu/branches/compare/$CURRENT..$LATEST
+ARG LIBGME_URL="https://bitbucket.org/mpyne/game-music-emu.git"
+ARG LIBGME_COMMIT=b3d158a30492181fd7c38ef795c8d4dcfd77eaa9
```

Comment:
Possible to use stable releases? seems like there are releases quite regular https://bitbucket.org/mpyne/game-music-emu/downloads/

Reply:
Can probably use this bump pipeline:
```sh
$ bump pipeline 'https://bitbucket.org/mpyne/game-music-emu.git|^0'
0.6.3
```

Reply:
Ok now when i checked closer quite regular was a bit of a exaggeration :) let's use master for now and see how open it updates. Maybe a comment why master?